src/twisted/words/test/test_irc.py

```diff
@@ -792,14 +806,14 @@ def test_support_CHANMODES(self):
         self._testFeatureDefault('CHANMODES', [('CHANMODES', 'b,,lk,ha,ha')])
 
         self.assertEqual(
-            self._parseFeature('CHANMODES', ''),
+            self._parseFeature('CHANMODES', ',,,'),
```

Comment:
Why is this change relevant to the 2->3 port?  These values really ought to be python-version independent...